bintray-release.sh

```diff
@@ -115,8 +115,18 @@ if [[ "$KONG_RELEASE" == "internal-preview" ]] ; then
     INTERNAL_PREVIEW=true
 fi
 
+if [[ "$KONG_RELEASE" == beta* ]] ; then
+    TAG_LATEST=
```

Comment:
Reminder for a future plan, these should be over loadable instead (or plus) of being a flag, so this kind of behavior can be tamed on run.